opentelemetry-sdk/src/trace/sampler/jaeger_remote/sampling_strategy.rs

```diff
@@ -107,9 +107,10 @@ impl Inner {
                 }
             })
             .unwrap_or_else(|_err| {
-                global::handle_error(TraceError::Other(
-                    "jaeger remote sampler mutex poisoned".into(),
-                ))
+                otel_warn!(
+                    name: "JaegerRemoteSampler.MutexPoisoned",
+                    message = "Unable to update Jaeger Remote sampling strategy: the sampler's internal mutex is poisoned, likely due to a panic in another thread holding the lock. The last known configuration will continue to be used until the application is restarted.",
```

Comment:
nit: also mention that no further attempts will be made to fetch config again.

Reply:
done